### ISSUE TITLE
Is it possible to not install dlib when installing this package?

### ISSUE THREAD

**Issue by abhisheksoni27** [ageitgey/face_recognition#133]
If you could only tell me how to go about it, it'd be great.

**Comment by ageitgey** [comment]
No, dlib is doing most of the hard work that makes this library function.

**Comment by ageitgey** [comment]
If you mean you already have dlib installed, you can use the no dependencies flag in pip to not force installing it.

**Comment by abhisheksoni27** [comment]
Works like charm. Thank you, Sir.
